draft-ietf-quic-load-balancers.md

```diff
@@ -752,20 +767,108 @@ such as congestion window, RTT, or MTU. Opaque data MAY also allow servers to
 distinguish between retry tokens (which trigger use of certain transport
 parameters) and NEW_TOKEN frame tokens.
 
+Some implementations of QUIC encode in the token the Initial Packet
+Number used by the client, in order to verify that the client sends the
+retried Initial with a PN larger that the triggering Initial. Such
+implementations will encode the Initial Packet
+Number as part of the opaque data.
+
+DISCUSS: we have a problem of intervals. We can either specify that the timestamp
```

Comment:
If it's the expiration time, then each token generator can specify its own token expiration time; it it's the start time, than the token consumer is always in control of the expiration. The difference is not huge. Consumer decisions happen later, and therefore have (slightly) more information. This is related to your DISCUSS point at Line 871; if the Retry service isn't checking timestamps, than it seems easier to configure if only the servers need to worry about the proper expiration interval.

I don't feel strongly either way.

Reply:
We talked and realized that as the NEW_TOKEN is also following this format, it is far better for the token to encode the expiration time so as not to force the Service to implement multiple policies and disambiguate the two types.